spec/spec_helper.rb

```diff
@@ -1,8 +1,8 @@
 # This file is copied to spec/ when you run 'rails generate rspec:install'
 
-require 'coveralls'
-Coveralls.wear!('rails') do
-  add_filter("/spec/")
+if ENV["TEST_SUITE"] == "vmdb"
```

Comment:
I don't think we don't want to register the `after(:suite)` unless we're on a CI server.